package.json

```
@@ -3,7 +3,7 @@
   "description": "Rapidly create HTML prototypes of GOV.UK services",
   "version": "13.13.6",
   "engines": {
-    "node": "^16.x || >= 18.x"
+    "node": "^18.x || >= 20.x"
```

@BenSurgisonGDS Need to stick v16 back in here

Probably worth setting all three to their first LTS release version?

This won't prevent other versions from running but npm will log a warning

```suggestion
    "node": "^16.13.0 || ^18.12.0 || ^20.9.0"
```

---

Sorry should have said, run `npm install` again and it'll be automatically updated in the shrinkwrap file too

(I think so anyway, it does for **package-lock.json**)